src/hooks/useActiveMenuStyles.js

```diff
@@ -10,15 +10,25 @@ const useActiveMenuStyles = () => {
   const location = useLocation();
   const theme = useTheme();
 
+  const headerActiveLinkStyle = {
+    color: theme.colors.menuTextActive,
+    fontWeight: theme.fontWeights.heading,
+    borderBottom: `2px solid ${theme.colors.primary}`,
+  };
+
+  const footerActiveLinkStyle = {
+    color: theme.colors.primary,
+  };
+
   const getActiveStyleForPathname = useCallback(
-    pathname => {
-      return isMatchingPathname(location.pathname, pathname)
-        ? {
-            color: theme.colors.menuTextActive,
-            fontWeight: theme.fontWeights.heading,
-            borderBottom: `2px solid ${theme.colors.primary}`,
-          }
-        : {};
+    (pathname, position) => {
+      let activeStyle =
```

Comment:
Why is activeStyle defined by `let` keyword? Let's use `const`:D

Reply:
Fixed 👍